Pak-Je-Kans.md

```diff
@@ -0,0 +1,47 @@
+# Wat wordt er gemaakt
+---
+We willen een plek maken op Mijn Amsterdam genaamd Pak Je Kans voorzieningencheck, hier kunnen mensen zien waar ze recht op hebben. Deze armoedevoorzieningen worden zo persoonlijk mogelijk gefilterd. Dit doen we met de gegevens die we van de burger hebben en krijgen, als de burger toegeeft dat we zijn of haar egevens gebruiken om deze voorzieningen te laten zien. Het doel van deze omgeving is ervoor zorgen dat de burger beter zicht heeft op wat ze allemaal recht hebben. De burgers kunnen het ook direct aanvragen.
```

Comment:
... `hier kunnen mensen zien waar ze recht op hebben.` ... Misschien kun je hier al wat specifieker zijn, het gaat om `voorzieningen voor mensen met een laag inkomen` Het woord `armoedevoorziening` kun je beter niet gebruiken.

...`als de burger toegeeft dat we zijn of haar egevens gebruiken`... moet zijn ...`als de burger toestemming geeft om zijn of haar gegevens te mogen gebruiken`...

`Het doel van deze omgeving is ervoor zorgen dat de burger beter zicht heeft op wat ze allemaal recht hebben. ` Deze zin is niet zo mooi, kun je die herschrijven?

`De burgers kunnen het ook direct aanvragen.` Wanneer en hoe?